bndtools.core/src/bndtools/wizards/project/NewBndProjectWizard.java

```diff
@@ -53,6 +55,13 @@ class NewBndProjectWizard extends AbstractNewBndProjectWizard implements ISkippi
 	public void init(IWorkbench workbench, IStructuredSelection currentSelection) {
 		super.init(workbench, currentSelection);
 
+		try {
+			// refresh, to not miss recently created project templates.
+			Central.refreshPlugins();
+		} catch (Exception e) {
+			throw Exceptions.duck(e);
+		}
```

Comment:
@pkriens just a note about why I have put it here: I'd prefer to put it at the end of `bndtools.wizards.project.AbstractNewBndProjectWizard.performFinish()` but I had (racy) issues that the new added Project needs to finished building before the repo-refresh worked for the `EclipseWorkspaceRepository`. 
This sometimes worked, sometimes not. E.g. when I had a debugger enabled then it mostly worked, because it had time to build. 

Question:
Do we have any mechanism to executed `Central.refreshPlugins();` after the new created project has been built?